src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int INDEX_OF_DATE = 0;
+    private static final int INDEX_OF_NAME = 1;
+    private static final int INDEX_OF_HOURS = 2;
+    private static final int INDEX_OF_SALARY_PER_HOUR = 3;
+    private static final String WHITE_SPACE = " ";
+    private static final String ROW = " - ";
+    private static final DateTimeFormatter FORMATTER
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate endDate = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder salaryInfo = new StringBuilder("Report for period "
+                + dateFrom + ROW + dateTo);
```

Comment:
FYI: It's better to create StringBuilder and then use the method append than concat string inside the constructor
```suggestion
        StringBuilder salaryInfo = new StringBuilder("Report for period ");
        salaryInfo.append(dateFrom).append(ROW).append(dateTo);
```
OR
```suggestion
        StringBuilder salaryInfo = new StringBuilder();
        salaryInfo.append("Report for period ").append(dateFrom)
            .append(ROW).append(dateTo);
```